doc/requirements.txt

```diff
@@ -26,3 +26,5 @@ sphinxcontrib-htmlhelp==2.0.0
 sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.5
+covalent-slurm-plugin
+covalent-ssh-plugin
```

Comment:
should we pin the versions for these plugins?

Reply:
Good q, Goal was to pin it only when having a stable releases. Until then, we want the latest version of plugins show up in latest version of rtd.

Reply:
I believe this will still not use the pre-release version.  As far as I know, the only way to do that is to specify the source repo in the requirements file.